src/SymbolServer.jl

```diff
@@ -27,6 +27,17 @@ mutable struct SymbolServerInstance
     end
 end
 
+function remove_non_general_pkgs!(pkgs)
```

Comment:
Could you add a test for this function? Any code that interacts with the internals of `Pkg` often needs to be adapted for different Julia versions, and for that it is really helpful to have a test that then runs during CI on all the different Julia versions we need to support.